TAT/Extensions/UIColor+TATColors.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIColor+TATColors.swift
+//  TAT
+//
+//  Created by Jamfly on 2019/2/24.
+//
+
+import UIKit.UIColor
+
+extension UIColor {
+
+  class func navigationBarPurple() -> UIColor {
```

Comment:
Could simply use `class var` if no calculation needed
``` swift
class var navigationBarPurple: UIColor {
  return UIColor(red: 113 / 255,
                 green: 108 / 255,
                 blue: 255 / 255,
                 alpha: 1)
}
```

Reply:
fix it and ready to merge.